src/main/java/hu/oe/nik/szfmv/automatedcar/model/utility/ModelCommonUtil.java

```diff
@@ -118,4 +129,27 @@ public static Position getBottomRightPoint(Position... points) {
 
         return p;
     }
+
+    /**
+     * @param topLeftX bal felső sarok X koordinátája
+     * @param topLeftY bal felső sarok Y koordinátája
+     * @param width négyszög szélessége
+     * @param height négyszög magasssága
+     * @return generált négyszög
+     */
+    public static Rectangle createRectangle(int topLeftX, int topLeftY, int width, int height) {
+        return new Rectangle(topLeftX, topLeftY, width, height);
+    }
+    
+    /**
+     * @param maxNumber
+     * @return 1 és maxNumber között ad vissza számokat
+     * @throws IllegalArgumentException, ha maxNumber < 1
+     */
+    public static int getRandom(int maxNumber) {
+        if (maxNumber < 1) {
+            throw new IllegalArgumentException("Max number must be greater than 1.");
```

Comment:
Consts? ERROR_MAX_NUMBER_GREATER_THAN_ONE pl.

Reply:
#151-ben megcsinálom, ne törődj ezzel

Reply:
Nem vagyok híve annak, hogy ide konstans kerüljön (nem is nagyon emlékszem, hogy láttam-e már ilyet), szándékosan csináltam így, de igazából mindegy, át is lehet írni.